src/components/FeaturePanel/Climbing/utils/photo.ts

```diff
@@ -11,6 +11,15 @@ export const removeFilePrefix = (name: string) => name?.replace(/^File:/, '');
 export const isWikimediaCommons = (tag: string) =>
   tag.startsWith('wikimedia_commons');
 
+export const isWikimediaCommonsPhoto = (tag: string) => {
+  // regexp to match wikimedia_commons, wikimedia_commons:2, etc. but not  wikimedia_commons:path, wikimedia_commons:whatever
+  const re = /^wikimedia_commons(:\d+)?$/;
```

Comment:
I recommend regexp with negative lookahead pattern - see here: https://github.com/zbycz/osmapp/blob/master/src/services/images/getImageDefs.ts#L90

Your regexp is missing out on `wikipedia_commons:something` tags.